src/citrine/resources/predictor.py

```diff
@@ -52,20 +54,50 @@ def _predictors_path(self, subpath: str, uid: Union[UUID, str] = None):
         path_template = self._path_template + (f"/{uid}" if uid else "") + f"/{subpath}"
         return format_escaped_url(path_template, project_id=self.project_id)
 
+    # Until we expose predictor versions in the SDK, and thus expose the
+    # difference between archving a predictor root and version, we should
+    # report the root's archive status.
+    def _inject_archive_info(self, predictor, is_archived=None):
```

Comment:
```suggestion
    def _inject_archive_info(self, predictor: Predictor, is_archived: bool = None) -> Predictor:
```
It's internal, but typehints are still convenient

Reply:
Fair enough.